library/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     compile project(':third_party:disklrucache')
     compile project(':annotation')
     compile "com.android.support:support-v4:${SUPPORT_V4_VERSION}"
-
+    compile "com.android.support:appcompat-v7:${SUPPORT_V7_VERSION}"
```

Comment:
Just read about provided dependencies
Indeed it looks like a good way to do this.